src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -901,7 +901,7 @@ class HookUtils(object):
                                (caller_name(), msg))
         # Update the resource usage information for each job
         if cgroup.cfg['periodic_resc_update']:
-            for jobid in event.job_list():
+            for jobid in event.job_list:
```

Comment:
Please add a comment above this line explaining that event.joblist is only iterable when the parenthesis are omitted.